src/main/java/by/exadel/internship/mapper/internship/InternshipMapper.java

```diff
@@ -42,4 +39,8 @@
     @IterableMapping(qualifiedByName = "internship")
     List<UserInternshipDTO> mapUserInternshipDTOList(List<Internship> internshipList);
 
+    @BeanMapping(nullValuePropertyMappingStrategy = NullValuePropertyMappingStrategy.IGNORE,
```

Comment:
Why do you ignore null value for mapping?

Reply:
remove it